src/components/Branding/Branding.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { BrandingStyles, BrandingContainer } from './styled-components';
+
+export default class Branding extends React.Component {
+  render() {
+    return (
+      <BrandingContainer style={{ width: 'inherit' }}>
```

Comment:
I'm not against inline styles for good reasons, this seems like you didn't need to do it inline.